### PR TITLE
Optimize Accessibility for Retrieve Test Result Screen (EXPOSUREAPP-4460)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Added screen reader support for the illustration on the "Retrieve Rest Result" screen when exploring it by touch.

To test this changes, activate TalkBack and open the "Have you been tested" Card on the Home Screen.

retrieve_test_result_content_description

### CODE REVIEW

**Review by SamuraiKek — APPROVED** [review]
At first I was testing it in English and it still said "Unlabelled", but then I realised we only had strings in German. Safe to say, the description works. Tested on Nokia 3 (Android 9).

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionDispatcherFragment.kt [hunk 1]
```diff
@@ -52,7 +52,7 @@ class SubmissionDispatcherFragment : Fragment(R.layout.fragment_submission_dispa
 
     override fun onResume() {
         super.onResume()
-        binding.submissionDispatcherRoot.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
+        binding.submissionDispatcherRoot.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionDispatcherFragment.kt:55]: Why is this change necessary?

**Review by SabineLoss — APPROVED** [review]
UA reviewed

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionDispatcherFragment.kt (hunk 1 above)
Comment (axelherbstreith) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionDispatcherFragment.kt:55]: During testing I found out that the component wasn't focused by default when using `TYPE_ANNOUCEMENT` and therefore nothing was announced by the screen reader. Also settings android:focusable="true"  alone didn't work out. Explicitly setting `TYPE_ANNOUCEMENT` to the view fixes the issue.

**Review by harambasicluka — APPROVED** [review]
